piker/clearing/_ems.py

```diff
@@ -1451,7 +1451,7 @@ async def _emsd_main(
     ) as (relay, feed, client_ready):
 
         brokerd_stream = relay.brokerd_stream
-        dark_book = _router.get_dark_book(broker)
+        dark_book = _router.get_dark_book(broker) 
```

Comment:
same thing here: trailing ws.

I'm not going to comment on any future ones presuming you'll clean it up in a new commit.